### PR TITLE
[SCP-482] feat: create get feed count method

### PR DESCRIPTION
**Pull request by maruamallo**
**Master branch** 
_Code Coverage Report Summary:_
  Classes: 91.51% (97/106)   
  Methods: 97.21% (453/466)  
  Lines:   92.97% (2724/2930)

**Current branch**
_Code Coverage Report Summary:_
  Classes: 91.67% (99/108)   
  Methods: 97.27% (464/477)  
  Lines:   93.12% (2790/2996)

### CODE REVIEW

File: src/Factory/Xml/Feed/FeedCountFactory.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+class FeedCountFactory
+{
+    public static function make(SimpleXMLElement $xml): FeedCount
+    {
+        self::validateStructure($xml, 'FeedCount');
+        self::validateStructure($xml->FeedCount, 'Total');
+        self::validateStructure($xml->FeedCount, 'Queued');
+        self::validateStructure($xml->FeedCount, 'Processing');
+        self::validateStructure($xml->FeedCount, 'Finished');
+        self::validateStructure($xml->FeedCount, 'Canceled');
```
Comment (lucasschlichting) [inline src/Factory/Xml/Feed/FeedCountFactory.php:20]: What about to create a const array of properties?
```php
  public const FEED_COUNT = 'FeedCount';
  public const PROPERTIES = [
    'Total',
    'Queued',
    'Processing',
    'Finished',
    'Canceled',
  ];
 public static function make(SimpleXMLElement $xml): FeedCount
    {
        self::validateStructure($xml, self::FEED_COUNT);
        foreach(self::PROPERTIES as property) {
           self::validateStructure($$xml->FeedCount, property);
        }
        ...
  }
  private static function validateStructure(SimpleXMLElement $xml, string $property): void
    {
         if (!property_exists($xml, $property)) {
            throw new InvalidXmlStructureException('FeedCount', $property);
        }
    }
```

File: src/Service/FeedManager.php [hunk 2]
```diff
@@ -180,4 +183,70 @@ public function getFeedList(): array
 
         return $feedsResponse;
     }
+
+    public function getFeedCount(): FeedCount
+    {
+        $action = 'FeedCount';
+        $requestId = uniqid((string) mt_rand());
```
Comment (lucasschlichting) [inline src/Service/FeedManager.php:190]: maybe gerenate the id like this?
```php
    return bin2hex(random_bytes(16));
```

File: src/Factory/Xml/Feed/FeedCountFactory.php (hunk 1 above)
Comment (maruamallo) [reply src/Factory/Xml/Feed/FeedCountFactory.php:20]: I like this! doing the change

File: src/Service/FeedManager.php (hunk 2 above)
Comment (maruamallo) [reply src/Service/FeedManager.php:190]: yes, we decided that today morning. Thanks

**Review by lucasschlichting — DISMISSED** [review]

**Review by ecastro98 — DISMISSED** [review]

**Pull request merged**